src/server/managers/AppListenerManager.ts

```diff
@@ -578,4 +583,24 @@ export class AppListenerManager {
             }
         }
     }
+
+    // Livechat
+    private async executeLivechatRoomClosed(data: ILivechatRoom): Promise<void> {
+        const cfLivechatRoom = Utilities.deepCloneAndFreeze(data);
+
+        for (const appId of this.listeners.get(AppInterface.ILivechatRoomClosedHandler)) {
+            const app = this.manager.getOneById(appId);
+
+            if (!app.hasMethod(AppMethod.EXECUTE_LIVECHAT_ROOM_CLOSED_HANDLER)) {
+                continue;
+            }
+
+            return app.call(AppMethod.EXECUTE_LIVECHAT_ROOM_CLOSED_HANDLER,
```

Comment:
```suggestion
            await app.call(AppMethod.EXECUTE_LIVECHAT_ROOM_CLOSED_HANDLER,
```

This `return` statement also prevents the execution of more than one app (I admit I hadn't noticed the one above 🙈). Would you agree with the proposed change?

Reply:
🥵